### PR TITLE
info_to_text

### PR DESCRIPTION
**Pull request by jvary**
Extra method to output 'info' to text variable, so we can output the info to logs.

### CODE REVIEW

**Review by DArdouin — APPROVED** [review]

**Comment by jvary** [comment]
Extra method to output 'info' to text variable, so we can output the info to logs

File: src/circuit_breaker.erl [hunk 1]
```diff
@@ -196,9 +197,17 @@ clear(Service) ->
...
+  Text = info_to_text(),
+  io:put_chars(Text).
+
+-spec info_to_text() -> io_lib:chars().
+%% @doc Print information regarding services.
+%% @end
+info_to_text() ->
+  Header  = format("Service", "Status", "Error", "Timeout", "CallTimeout"),
+  Split   = format(dup(31, $-), dup(15, $-), dup(8, $-), dup(8, $-), dup(8, $-)),
+  Entries = ets:foldr(fun do_info/2, [], ?TABLE),
```
Comment (mikpe) [inline src/circuit_breaker.erl]: The ets table isn't `ordered_set` so calling `foldr/3` doesn't have the effect implied by its name. Please change that back to `foldl/3`.
Please also squash to a single commit.

Comment (jvary) [reply src/circuit_breaker.erl]: @mikpe : comments resolved

**Review by mikpe — APPROVED** [review]

**Review by jesperes — APPROVED** [review]

**Comment by mikpe** [comment]
Merged and tagged as 1.0.2

**Pull request merged**